prow/cluster/components/hock-deck_ingress.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
There is a typo in the file name '`hock`'.

Reply:
fixing.